.github/workflows/deployments.yml

```diff
@@ -1,5 +1,11 @@
 on:
   workflow_dispatch:
+    inputs:
+      update_registry_cache:
```

Comment:
This was an oversight - there was no way to manually push a build cache update to the registry. I randomly added it here but will make a separate PR if desired.